packages/core/strapi/lib/middlewares/session/index.js

```diff
@@ -1,33 +1,46 @@
 'use strict';
 
 const path = require('path');
-const _ = require('lodash');
+const { defaultsDeep, isEmpty, get, isObject, has, isString, assign } = require('lodash/fp');
 const session = require('koa-session');
 
+const defaults = {
+  enabled: true,
+  client: 'cookie',
+  key: 'strapi.sid',
+  prefix: 'strapi:sess:',
+  ttl: 864000000,
+  rolling: false,
+  secretKeys: ['mySecretKey1', 'mySecretKey2'],
```

Comment:
These really need to be auto-generated, we never did it in v3 but it's a security risk